k8s-daemonset/k8s/linkerd-cni.yml

```diff
@@ -0,0 +1,128 @@
+# runs linkerd in a daemonset, in linker-to-linker mode
+# for CNI setups such as Calico and Weave
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: l5d-config
+data:
+  config.yaml: |-
+    admin:
+      port: 9990
+
+    namers:
+    - kind: io.l5d.k8s
+      experimental: true
+      host: localhost
+      port: 8001
+
+    telemetry:
+    - kind: io.l5d.prometheus
+    - kind: io.l5d.recentRequests
+      sampleRate: 0.25
+
+    usage:
+      orgId: linkerd-examples-daemonset
```

Comment:
Can you set this to `linkerd-examples-daemonset-cni`? That more consistent with the rest of the configs in this directory.